plugins/parsers/xpath/README.md

```diff
@@ -13,11 +13,51 @@ For supported XPath functions check [the underlying XPath library][xpath lib].
 | [Extensible Markup Language (XML)][xml] | `"xml"`               |         |
 | [JSON][json]                            | `"xpath_json"`        |         |
 | [MessagePack][msgpack]                  | `"xpath_msgpack"`     |         |
-| [Protocol buffers][protobuf]            | `"xpath_protobuf"`    | [see additional parameters](#protocol-buffers-additional-settings)|
+| [Protocol-buffers][protobuf]            | `"xpath_protobuf"`    | [see additional parameters](#protocol-buffers-additional-settings)|
 
-### Protocol buffers additional settings
+### Protocol-buffers additional settings
 
-For using the protocol-buffer format you need to specify a protocol buffer definition file (`.proto`) in `xpath_protobuf_file`, Furthermore, you need to specify which message type you want to use via `xpath_protobuf_type`.
+For using the protocol-buffer format you need to specify additional (*mandatory*) properties for the parser. Those options are described here.
+
+#### `xpath_protobuf_file` (mandatory)
+
+Use this option to specify the name of the protocol-buffer definition file (`.proto`).
+
+#### `xpath_protobuf_type` (mandatory)
+
+This option contains the top-level message file to use for deserializing the data to be parsed. Usually, this is constructed from the `package` name in the protocol-buffer definition file and the `message` name as `<package name>.<message name>`.
+
+#### `xpath_protobuf_import_paths` (optional)
+
+In case you import other protocol-buffer definitions within your `.proto` file (i.e. you use the `import` statement) you can use this option to specify paths to search for the imported definition file(s). By default the imports are only searched in `.` which is the current-working-directory, i.e. usually the directory you are in when starting telegraf.
+
+Imagine you do have multiple protocol-buffer definitions (e.g. `A.proto`, `B.proto` and `C.proto`) in a directory (e.g. `/data/my_proto_files`) where your top-level file (e.g. `A.proto`) imports at least one other definition
```

Comment:
Does this path need to be absolute? If it is relative, does it need to be relative to the telegraf binary?

If the files are all in the same directory with the binary do you need to specify the full path or just '.'?